bug-bounty-hof/web.yml

```diff
@@ -1,4 +1,54 @@
 names:
+- name: endang
+  date: 2022-06-27
+- name: redparsec
+  date: 2022-06-13
+  twitter: "https://twitter.com/r3dpars3c"
+  url: https://twitter.com/https://twitter.com/r3dpars3c
```

Comment:
The redparsec entry is more broken than it was in #107: the URL has two "https://" in it, and the `twitter:` line is still a URL instead of a @handle.

Is this the same person as on line 9 of the client.yml diff above? Same @r3dpars3c twitter handle, but up there the name is "Anonymous" and here it's "redparsec"

Reply:
Yes; I've corrected it in a new commit. This reporter has changed email and credit in the past so I think the script was slightly out of date from those corrections.